config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main.yaml

```diff
@@ -198,19 +198,19 @@ periodics:
         - name: GINKGO_FOCUS
           value: "Workload cluster creation"
         - name: GINKGO_SKIP
-          value: ""
+          value: "AKS cluster"
```

Comment:
Should we add a well-known tag for AKS tests in the CAPZ spec names similar to `[Conformance]`, `[Required]`, etc. instead of relying on the string containing this substring?

Reply:
How about this?: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2777/files

Reply:
That works for me 👍 @mboersma @nojnhuh what do y'all think?